project/Settings.scala

```diff
@@ -90,22 +91,33 @@ object Settings {
       addCompilerPlugin("com.olegpy" %% "better-monadic-for" % "0.3.0")
     )
 
+  lazy val entrypoint = "/opt/docker/bin/entrypoint.sh"
+
   lazy val commonDockerSettings = List(
     maintainer := "workbench-interactive-analysis@broadinstitute.org",
     dockerBaseImage := "oracle/graalvm-ce:1.0.0-rc16",
     dockerRepository := Some("us.gcr.io"),
     dockerExposedPorts := List(8080),
     dockerUpdateLatest := true,
+    // See https://www.scala-sbt.org/sbt-native-packager/formats/docker.html#add-commands
     dockerCommands ++= List(
-      ExecCmd("CMD", "export", "CLASSPATH=lib/*jar")
+      // Change the default umask for welder to support R/W access to the shared volume
+      Cmd("USER", "root"),
+      ExecCmd("RUN", "/bin/bash", "-c", s"echo '#!/bin/bash\\numask 002; /opt/docker/bin/server' > $entrypoint"),
+      Cmd("RUN", s"chown -R welder-user:users /opt/docker && chmod u+x,g+x $entrypoint"),
```

Comment:
Does `g` need permission to execute welder?

Reply:
No but I tried to be consistent — the sbt plugin also grants u+x,g+x to the /opt/docker/bin/server executable